index.js

```diff
@@ -0,0 +1,199 @@
+import { arrSliderText } from "./sliderText.js";
+const containerHeaderMenu = document.getElementsByClassName(
```

Comment:
add empty line between imports and constants, it is more readable

Reply:
done